include/SFML/System/Angle.inl

```diff
@@ -0,0 +1,258 @@
+////////////////////////////////////////////////////////////
+//
+// SFML - Simple and Fast Multimedia Library
+// Copyright (C) 2007-2022 Laurent Gomila (laurent@sfml-dev.org)
+//
+// This software is provided 'as-is', without any express or implied warranty.
+// In no event will the authors be held liable for any damages arising from the use of this software.
+//
+// Permission is granted to anyone to use this software for any purpose,
+// including commercial applications, and to alter it and redistribute it freely,
+// subject to the following restrictions:
+//
+// 1. The origin of this software must not be misrepresented;
+//    you must not claim that you wrote the original software.
+//    If you use this software in a product, an acknowledgment
+//    in the product documentation would be appreciated but is not required.
+//
+// 2. Altered source versions must be plainly marked as such,
+//    and must not be misrepresented as being the original software.
+//
+// 3. This notice may not be removed or altered from any source distribution.
+//
+////////////////////////////////////////////////////////////
+
+namespace priv
+{
+    constexpr float pi = 3.141592654f;
+    constexpr float two_pi = 2 * pi;
```

Comment:
There's a name for this: `tau`

Tau has become the de-facto name in the programming world. From [Wikipedia](https://en.wikipedia.org/wiki/Turn_(angle)#Tau_proposals):

> In 2012, the educational website Khan Academy began accepting answers expressed in terms of τ.
In June 2017, for release 3.6, the Python programming language adopted the name tau to represent the number of radians in a turn.
The τ-functionality is made available in the Google calculator and in several programming languages like Python, Raku, Processing, Nim, and Rust.
It has also been used in at least one mathematical research article,[41] authored by the τ-promoter Peter Harremoës.
In 2020, for release 5.0, Tau was added to .NET Core (which is being rebranded as ".NET" for the 5.0 release).

Reply:
Ah yes I forgot about tau. I'll fix that.